projects/1.16/assets/tinkers-archery/tinkersarchery/lang/zh_cn.json

```
@@ -0,0 +1,247 @@
+  "material.tinkersarchery.silky_cloth.arrow": "箭向上飞一会儿，然后坠落于地面。",
+  "material.tinkersarchery.silky_cloth.bow": "箭稍微向上射出以帮助瞄准。",
+  "material.tinkersarchery.silky_cloth.flavor": "穿在身上非常柔软",
+  "material.tinkersarchery.silky_cloth.weight": "重量影响箭飞升和下落的速度。",
+  "material.tinkersarchery.slime": "史莱姆",
+  "material.tinkersarchery.slime.bow": "在完全黏液覆层时,弓的拉伸速度和拉伸重量增加了0.5。",
+  "material.tinkersarchery.slime.flavor": "蹦蹦跳跳",
+  "material.tinkersarchery.slime_leaf": "史莱姆叶",
+  "material.tinkersarchery.slime_leaf.arrow": "箭在空中弹射5次。",
+  "material.tinkersarchery.slime_leaf.flavor": "这绝对是一种粘粘的木材。",
+  "material.tinkersarchery.slime_leaf.weight": "较轻的箭下落得较慢，较稳定的箭能较长时间保持高速。",
+  "material.tinkersarchery.steel_wire": "钢丝",
+  "material.tinkersarchery.steel_wire.bow": "箭总是不稳定的。",
+  "material.tinkersarchery.steel_wire.flavor": "粘土比不上钢丝的力量!",
+  "material.tinkersarchery.tantalum": "钽",
+  "material.tinkersarchery.tantalum.bow": "弓的准确度每级提升40%。",
+  "material.tinkersarchery.tantalum.encyclopedia": "每级对生命值低于一半的敌人增加25%的伤害。",
+  "material.tinkersarchery.tantalum.flavor": "射箭钽准啦",
+  "modifier.tinkersarchery.accurate": "精确",
+  "modifier.tinkersarchery.accurate.description": "弓更精准",
+  "modifier.tinkersarchery.accurate.flavor": "不往你指的地方不射的弓有什么用?",
+  "modifier.tinkersarchery.airborne": "破空",
+  "modifier.tinkersarchery.airborne.description": "用弓射击时，你的速度不再增加到箭上",
+  "modifier.tinkersarchery.airborne.flavor": "飞向天空",
+  "modifier.tinkersarchery.antigravity_trajectory": "反重力轨迹",
+  "modifier.tinkersarchery.antigravity_trajectory.description": "箭不服从重力",
+  "modifier.tinkersarchery.antigravity_trajectory.flavor": "零重力",
+  "modifier.tinkersarchery.autoaim": "自动瞄准",
+  "modifier.tinkersarchery.autoaim.description": "箭指向最近的实体",
+  "modifier.tinkersarchery.autoaim.flavor": "我从不失手",
+  "modifier.tinkersarchery.blazing": "炽热",
+  "modifier.tinkersarchery.blazing.description": "箭着火了",
+  "modifier.tinkersarchery.blazing.flavor": "我火了！",
+  "modifier.tinkersarchery.bouncing_trajectory": "弹跳轨迹",
+  "modifier.tinkersarchery.bouncing_trajectory.description": "箭在地面上弹跳",
+  "modifier.tinkersarchery.bouncing_trajectory.flavor": "跳一跳",
+  "modifier.tinkersarchery.burst": "迸发",
+  "modifier.tinkersarchery.burst.description": "下次你连续拉弓时，它会拉得更快",
+  "modifier.tinkersarchery.burst.flavor": "每一次击杀都会带来荣誉",
+  "modifier.tinkersarchery.chaining": "连环",
+  "modifier.tinkersarchery.chaining.attack_damage": "连锁伤害",
+  "modifier.tinkersarchery.chaining.description": "杀死敌人后，对下一个敌人造成成倍的伤害",
+  "modifier.tinkersarchery.chaining.flavor": "连杀奖励",
+  "modifier.tinkersarchery.clean": "彻底",
+  "modifier.tinkersarchery.clean.accuracy": "尝尝完全拉伸的精确度",
+  "modifier.tinkersarchery.clean.description": "你的弓越拉得紧，箭就射得越准",
+  "modifier.tinkersarchery.clean.flavor": "简单",
+  "modifier.tinkersarchery.finishing": "终结",
+  "modifier.tinkersarchery.finishing.attack_damage": "半血下的伤害",
+  "modifier.tinkersarchery.finishing.description": "对血量低的敌人造成更多伤害",
+  "modifier.tinkersarchery.finishing.flavor": "朽木凋零 / 寒风刺骨 / 命染黄沙",

```

```suggestion
  "modifier.tinkersarchery.finishing": "终结",
  "modifier.tinkersarchery.finishing.attack_damage": "低于半血时伤害",
  "modifier.tinkersarchery.finishing.description": "对血量低的敌人造成更多伤害",
  "modifier.tinkersarchery.finishing.flavor": "落叶正飘散/雪花飞舞落至地/汝命已凋零",
```
无主之地，零，原文为五七五音节排布，应为俳句
大致依照https://tieba.baidu.com/p/3572754813 修改了一下（~~完全找不到对应的中文译法~~）